### PR TITLE
Test queue media

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: tests/media.py [hunk 1]
```diff
@@ -145,6 +146,42 @@ def media_exposed_time(cfg):
...
+    medias = [m.filename for m in cfg.medias if m.filename.endswith(("mp4", "jpg"))]
+
+    ag = AutoGrid(range(nb_videos))
+    for video_id, _, col, pos in ag:
+        start = video_id * cfg.duration / nb_videos
+        animkf = [
+            ngl.AnimKeyFrameFloat(start, 0),
+            ngl.AnimKeyFrameFloat(start + cfg.duration, cfg.duration),
+        ]
+        m = ngl.Media(medias[video_id % len(medias)], time_anim=ngl.AnimatedTime(animkf))
```
Comment (mbouron-gpsw) [inline tests/media.py:166]: nit; `media` ?

File: tests/media.py [hunk 2]
```diff
@@ -145,6 +146,42 @@ def media_exposed_time(cfg):
...
+
+        t = ngl.Texture2D(data_src=m, min_filter="linear", mag_filter="linear")
```
Comment (mbouron-gpsw) [inline tests/media.py:168]: nit: `texture` ?

File: tests/media.py [hunk 3]
```diff
@@ -145,6 +146,42 @@ def media_exposed_time(cfg):
     return render
 
 
+@test_fingerprint(width=1024, height=1024, nb_keyframes=30, tolerance=1)
+@scene(overlap_time=scene.Range(range=[0, 10], unit_base=10), dim=scene.Range(range=[1, 10]))
+def media_queue(cfg, overlap_time=7.0, dim=3):
+    cfg.duration = 10
+    cfg.aspect_ratio = (1, 1)
+    tqs = []
```
Comment (mbouron-gpsw) [inline tests/media.py:154]: what `tqs` means ?

File: tests/media.py (hunk 1 above)
Comment (cboesch-gpsw) [reply tests/media.py]: renamed

File: tests/media.py (hunk 2 above)
Comment (cboesch-gpsw) [reply tests/media.py]: renamed

File: tests/media.py (hunk 3 above)
Comment (cboesch-gpsw) [reply tests/media.py]: probably time queues (name used in the example scene); renamed to `queued_medias`

**Review by mbouron-gpsw — APPROVED** [review]

**Pull request merged**